contentcuration/contentcuration/utils/automation_manager.py

```diff
@@ -0,0 +1,52 @@
+from contentcuration.utils.recommendations import RecommendationsAdapter
+from contentcuration.utils.recommendations import RecommendationsBackendFactory
+
+
+class AutomationManager:
+    def __init__(self):
+        self.reccomendations_backend_factory = RecommendationsBackendFactory()
+        self.reccomendations_backend_instance = self.reccomendations_backend_factory.create_backend()
```

Comment:
perhaps `recomendations_backend_instance` and `recomendations_backend_factory` instead?